doc/source/examples/dem/bouncing-particle/bouncing-particle.rst

```diff
@@ -51,24 +51,24 @@ The ``grid type`` in this example is a ``hyper_cube``. Its dimensions are 2.0 m
 Insertion Info
 ~~~~~~~~~~~~~~~~~~
 
-Since the insertion of the particle must be done at as specific height, the ``list`` insertion method is used. The ``insertion frequency`` can be set to any value, since we're only using one particle. The particle is at a height of 0.5 m in the center of the X,Y plane.
+Since the insertion of the particle must be done at as specific height, the ``list`` insertion method is used. The ``insertion frequency`` can be set to any value, since we're only using one particle. The insertion location is centered in the X-Y plane and at a height of 0.5 m in the Z direction.
```

Comment:
```suggestion
Since the insertion of the particle must be done at a specific location, the ``list`` insertion method is used. The ``insertion frequency`` can be set to any value, since we're only using one particle. The insertion location is centered in the X-Y plane and at a height of 0.5 m in the Z direction.
```